### PR TITLE
day24

### PR DESCRIPTION
**Pull request by Nejilee**

### CODE REVIEW

File: Neji Lee/DAY 24 - 54/day 24.md [hunk 1]
```diff
@@ -8,10 +8,14 @@
 >> POD, Nodes, also got to know more about differnent components of K8s architecture 
   >> Control plane, Kube schedular, Kubernetes API-service, control manager, etcd, kube-proxy, kubelet and docker engine
```
Comment (kranurag7) [inline Neji Lee/DAY 24 - 54/day 24.md:9]: *controller-manager*

File: Neji Lee/DAY 24 - 54/day 24.md [hunk 2]
```diff
@@ -8,10 +8,14 @@
...
+
+2. Studied about docker volume and how to share it, topics covered
 >> What is docker volume/
 >> Different ways of creating docker volume - through Dockerfile and through command.
 >> How to share this volume among different containers?
 >> Mapping root host directory to root docker directory.
 >> Benefits of docker volume
 >> Different cmds to rm, inspect and ls docker volume 
+
+3.  Practised some concepts in Gloang
```
Comment (kranurag7) [inline Neji Lee/DAY 24 - 54/day 24.md:20]: *Golang*

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Comment by kranurag7** [comment]
@Nejilee Good luck for the LFX mentorship. Keep learning and moving ahead.

**Comment by Nejilee** [comment]
> @Nejilee Good luck for the LFX mentorship. Keep learning and moving ahead.

Thank you so much. Hope my application will get selected.

**Pull request merged**